src/Login.jsx

```diff
@@ -15,15 +15,7 @@ class Login extends Component {
   }
   render() {
     return (
-      <div
-        className={`login-page`}
-        // style={{
-        //   left: this.props.loginVisibility
-        //     ? "calc(100%/2 - 180px)"
-        //     : "calc(100%/2 - 1800px)",
-        //   opacity: this.props.loginVisibility ? 1 : 0,
-        // }}
-      >
+      <div className={`login-page`}>
```

Comment:
When it's a single string you should pass it like this: 
`className="login-page"`